homeassistant/components/statistics/sensor.py

```diff
@@ -112,8 +138,7 @@ def async_stats_sensor_state_listener(entity, old_state, new_state):
             )
 
             self._add_state_to_queue(new_state)
-
-            self.async_schedule_update_ha_state(True)
+            self.hass.async_create_task(self._debounced_updater.async_call())
```

Comment:
just await it directly. No need to create a task. Requires you to change `@callback` -> `async def`

Reply:
Done